packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -74,11 +78,21 @@ export class WaferMap extends FoundationElement {
     @attr({ attribute: 'color-scale-mode' })
     public colorScaleMode: WaferMapColorScaleMode = WaferMapColorScaleMode.linear;
 
+    /**
+     * @internal
+     */
+    public worker!: Remote<MatrixRenderer>;
+
     /**
      * @internal
      */
     public readonly canvas!: HTMLCanvasElement;
 
+    /**
+     * @internal
+     */
+    public readonly workerCanvas!: HTMLCanvasElement;
```

Comment:
I still don't know why we are exposing both `canvas` and `workerCanvas` on the public API. Why is there a need for two distinct canvas elements? If there are different setup requirements for the element between the two scenarios, can't we just perform the appropriate setup as needed when we switch between them?

Reply:
I was preparing an explanation for this, when sharing the same canvas both wafers, stable and experimental, stop working, not sure why, I'll give an update

Reply:
Meanwhile I managed to use only one canvas